bundle/manifests/kueue-operator.clusterserviceversion.yaml

```diff
@@ -74,7 +74,6 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-kueue-operator
     operators.openshift.io/valid-subscription: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.33.0
```

Comment:
You might need to update this script as well: https://github.com/openshift/kueue-operator/blob/main/hack/preserve-bundle-labels.sh#L58